app/src/main/java/com/hadiyarajesh/notex/utility/ComposeSingleClickable.kt

```diff
@@ -0,0 +1,58 @@
+package com.hadiyarajesh.notex.utility
+
+import androidx.compose.foundation.LocalIndication
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.interaction.MutableInteractionSource
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.composed
+import androidx.compose.ui.platform.debugInspectorInfo
+import androidx.compose.ui.semantics.Role
+
+internal interface MultipleEventsCutter {
```

Comment:
This is for which functionality?

Reply:
and where will we use it?

Reply:
For now I have used when user click on the reminder item.